packages/main/src/pages/static/contact-us/components/ContactSideBar.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import contactStyle from "./ContactSideBar.module.css";
+import MessageIcon from "../assets/message-icon.svg";
+import LocationIcon from "../assets/location-icon.svg";
+import PhoneIcon from "../assets/phone-icon.svg";
+
+import FacebookIcon from "../assets/facebook-icon.svg";
+import TwitterIcon from "../assets/twitter-icon.svg";
+import InstagramIcon from "../assets/instagram-icon.svg";
+import LinkedInIcon from "../assets/linkedin-icon.svg";
```

Comment:
We have icon libraries already installed
why are you manually adding svgs?

Reply:
I couldn't find the exact same icons on the icon libraries.

Reply:
`React-icons` has all these icons. Its really not necessary to have svgs for this

Reply:
Will work on that now.